enterprise/backup/src/main/java/org/neo4j/backup/BackupProtocolService.java

```diff
@@ -109,8 +107,8 @@
 
     BackupProtocolService( OutputStream logDestination )
     {
-        this( DefaultFileSystemAbstraction::new, FormattedLogProvider.toOutputStream( logDestination ), logDestination,
-                new Monitors(), null );
+        this( DefaultFileSystemAbstraction::new, FormattedLogProvider.toOutputStream( logDestination ), logDestination, new Monitors(),
```

Comment:
The other constructor has `pageCache` annotated with `@Nullable` which isn't true any more.

Reply:
Addressed